### DISCUSSION TITLE
Not getting confirmation email and can't change email

### DISCUSSION THREAD

**Discussion by hqaddomi** [mastodon/mastodon#20072] [Q&A] [answered]
Hey there folks,

I registered for Mstodon.sdf.org but I did not get an email confirmation. I tried resending many times with no luck. I tried even changing the email address on my account, it says the changes were saved but they're not and now I'm stuck! Can't verify my account or recreate it with the same username. 

Can anybody help?

**Comment by MrGeneration** [comment]
The domain `mstodon.sdf.org` doesn't seem to be responding right now.
All you can do is try to contact the admin by email.

The email sendout process highly depends on the instance and is probably nothing anyone here can help you with.
That is if you already checked your spam folder and used a valid, functioning mail address. :-)

**Reply by Adagio4639** [reply]
Ok, this appears to be an ongoing problem. It's December 16 and I'm going through the very same issue. I'm not getting the confirmation email. These comments are from Nov. This is the middle of December, and the same problem exists. I'd like to dump Twitter and move to this platform, but this doesn't inspire a lot of confidence in the site.

**Reply by Adagio4639** [reply]
this is very weird. the system is recognizing my username as if my account exists. But I can't login using my email and password even though the account exists.

**Reply by MrGeneration** [reply]
Okay. If you'd like you can try my instance: `xxxx`.
If your account is at fault I can tell you. ;(

**Reply by Adagio4639** [reply]
This is what I get. 
If your email address exists in our database, you will receive an email with instructions for how to confirm your email address in a few minutes. Please check your spam folder if you didn't receive this email.
I checked the spam folder. NOTHING.  As you can see, my username, Adagio4639 is working. I did the normal signup process. Email, username, password. but no confirmation email from Mastodon.

**Reply by mohe2015** [reply]
On which server instance did you register? It is highly likely that the particular server instance is unreliable. As Mastodon is federated there are multiple instances to choose from.

**Comment by Adagio4639** [comment]
I have no idea what you're even talking about. What do you mean by federated servers? I simply opened an account with my email address and username and password. Then I submitted it and I was to receive a confirmation email, which I haven't received.  I have no idea what server I registered with or whether its reliable or not. I've never encountered anything that is as complicated as this, just to open an account.

**Reply by mohe2015** [reply]
There is not a single Mastodon server. There are lots of and it depends on which you registered on. Which website did your browser show in the url bar?

**Reply by Adagio4639** [reply]
What does Git Hub have to do with any of this? As you can see, this problem has been ongoing since last Nov. This is the Middle of December and there are still people encountering the very same problem. Send the damn confirmation email as we are being told. Why is this such a fucking problem??

**Reply by mohe2015** [reply]
There are good reasons nobody has created a second twitter yet and one of them is that its extremely expensive to host such a massive server that it can handle billions of users.

**Reply by Adagio4639** [reply]
I'm using Microsoft Edge. that's what I registered on.

**Reply by mohe2015** [reply]
Can you tell us what the highlighted part (the url bar) shows when you are on the Mastodon server's login or registration page?
![2022-12-16_18-39](https://user-images.githubusercontent.com/13287984/208156677-ada6a5c8-a583-448a-8212-e61f4f97c479.png)

**Comment by nightpool** [comment, marked as answer]
This forum is for the development and maintenance of the Mastodon server software. This is a free software package that can be run by anyone who wants to provide their own, connected, federated social media software. It is not a website, just code that others use to run a website.

The people who maintain this software are, for the most part, not involved or associated with the individual websites that people run or sign up on. You'll have to contact the person running your server to get more details about what may or may not be happening here.

**Comment by Adagio4639** [comment]
https://joinmastodon.org/servers

L

**Reply by mohe2015** [reply]
> https://joinmastodon.org/servers L

You don't know any more which of these you chose? Because that is pretty relevant.

Anyways if it didn't work you can also just choose another server. Unfortunately the official instance by the maintainers of this software https://mastodon.social/ is not accepting registrations currently. So you probably need to choose one from the page you linked that looks nice to you. If it causes issues you can always choose another one or try to contact it's admin which is listed on the about page of the server.

**Comment by Adagio4639** [comment]
These are the URL's..
https://mastodon.sdf.org/explore
https://soziale.cloud/auth/sign_in
https://joinmastodon.org/servers

What does GitHub have to do with this?

I don't understand why I'm told that I'm sent a confirmation email, only to find out that I wasn't sent one and others haven't gotten one dating back to Nov 7th.  What completely pisses me off is being told something is sent to me and if I don't receive it, then have it resent, and I still don't get it. Then I'm told about servers. What does that have to do with being told that I'm being sent a confirmation email to confirm my account?

L

**Comment by Adagio4639** [comment]
How am I supposed to know what server was being used?  I provided the URL to the site to sign up with. Then I provided the URL's for the confirmation which I didn't receive. Then I provided another URL. I have no idea what server I'm on or where to find that out. Why is this so complicated? There's a thing we call the KISS rule. Keep It Simple Stupid. Your system is requiring people to jump through a lot of hoops.

L

**Comment by Adagio4639** [comment]
How do I know what server I used? I just went to the URL for Mastodon and signed up.

**Comment by Adagio4639** [comment]
I think I see how this works. There are a bunch of servers that you can apply to, that cover specific topics. Is that right? There doesn't appear to be a server that focuses on politics.

**Reply by mohe2015** [reply]
Yes that is right. You can also follow people on other servers (so choosing a server is mostly important for where your data is). I quickly looked and also couldn't find one that's for politics specifically. You could probably choose one from the `General` category. You should probably check that the server has >1k members or so (shown on the start page on the left) so there is less probability of issues with the server.

**Reply by mohe2015** [reply]
You could also set the "Legal structure" at the top of the server search page to "Public organization" if you want

**Reply by mohe2015** [reply]
Also all servers that say "Apply for an account" need to confirm you manually so you can also get rejected. It's probably easier to choose one that says "Create account"

**Comment by nightpool** [comment]
This discussion thread is significantly off-topic at this point and it doesn't have any replies from the original user @hqaddomi, so i'm going to be locking it now. @Adagio4639, if you're still having issues, you're welcome to start a new thread or converse with @mohe2015 over email.
